packages/fluentui/react-northstar/src/components/Alert/Alert.tsx

```diff
@@ -96,10 +111,12 @@ export interface AlertProps extends UIComponentProps, ContentComponentProps<Shor
   body?: ShorthandValue<BoxProps>;
 }
 
-export interface AlertState {
-  visible: boolean;
-  bodyId: string;
-}
+export type AlertStylesProps = Pick<
```

Comment:
```suggestion
export type AlertStylesProps = Required<Pick<
```

Reply:
This will force to pass all props to `mapPropsToStyles`

Reply:
Please use this type in `alertStyles` and `teams/types.ts`

Reply:
And then, please check. May be some props will be not required for styles function 🤔

Reply:
Done

Reply:
Please add fix type there `packages/fluentui/react-northstar/src/themes/teams/components/Alert/alertStyles.ts`:

```diff
-const alertStyles: ComponentSlotStylesPrepared<AlertProps, AlertVariables> = {
+const alertStyles: ComponentSlotStylesPrepared<AlertStylesProps, AlertVariables> = {
```

And there `packages/fluentui/react-northstar/src/themes/teams/types.ts`:

```diff
-  Alert: AlertProps;
+  Alert: AlertStylesProps;
```

Reply:
done